teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization = 'Unknown')
+    super(age, name, parent_permission)
+    @specialization = specialization
+  end
```

Comment:
- [ ] I am not able to create teachers, kindly update your initialize method to match the number of arguments passed when creating a teacher
![image](https://user-images.githubusercontent.com/66097371/192822919-bb6801b8-9261-4b88-a1fe-4e25dfe98fc8.png)

Reply:
oh great job thank you so much i have fix it all kudos to you